bundles/net.tourbook/src/net/tourbook/messages.properties

```diff
@@ -1731,6 +1731,7 @@ Pref_Appearance_Check_AutoOpenTagging                = &Open "Add Tags..." autom
 Pref_Appearance_Check_TaggingAnimation               = Show &animation  \u00BB\u00BB\u00BB\u00BB   in the context menu
 Pref_Appearance_Dialog_ResetAllToggleDialogs_Message = All hidden dialogs will be displayed again.
 Pref_Appearance_Dialog_ResetAllToggleDialogs_Title   = Hidden Dialogs
+Pref_Appearance_Group_PaceDisplay                    = Pace Display Menu
 Pref_Appearance_Group_Tagging                        = Tagging Context Menu
```

Comment:
I don't understand the name `Pace Display Menu`, is there a pace display menu?

Reply:
No, I will fix that

Reply:
Fixed